tests/main/sbuild/task.yaml

```diff
@@ -0,0 +1,21 @@
+summary: Ensure snapd builds correctly in sbuild
+
+# takes a while
+priority: 500
+
+systems: [debian-sid-*]
+
+execute: |
+    echo "Ensure we have sbuild"
+    apt install -y sbuild eatmydata
+
+    echo "Create a sid sbuild env"
+    eatmydata sbuild-createchroot --include=eatmydata,ccache,gnupg sid /srv/chroot/sid-amd64-sbuild http://deb.debian.org/debian
+
+    echo "Allow test user to run sbuild"
+    sbuild-adduser test
+
+    echo "And build it normally"
```

Comment:
We should eventually document why this happens but I'm happy just having this so I don't even want to sound like I want to block it :)